cmake/scripts/windows/ArchSetup.cmake

```diff
@@ -1,5 +1,5 @@
 # Minimum SDK version required to build
-set(VS_MINIMUM_BUILD_SDK_VERSION 10.0.22621.0)
+set(VS_MINIMUM_BUILD_SDK_VERSION 10.0.26100.0)
```

Comment:
What about UWP, bump as well?

Reply:
Changes in this PR only affects Windows desktop, so there is no reason to require increaser SDK for UWP/Xbox